helm/cluster-cloud-director/Chart.yaml

```diff
@@ -23,7 +23,7 @@ restrictions:
     - cloud-director
 dependencies:
   - name: cluster
-    version: "1.4.1"
+    version: "1.5.1"
```

Comment:
this version contains all the required systemd options which were added to support this chart